src/components/DropdownProblem.tsx

```diff
@@ -59,8 +59,8 @@ export const DropdownProblem = ({
       setFeedback(determineFeedback(values.response, encourageResponse, answerResponses, evaluateInput))
       allowedRetryCallback()
     } else {
+      setFeedback(attemptsExhaustedResponse)
```

Comment:
Minor clarification question: Is there a specific reason why the relative ordering of this invocation was moved to come before `exhaustedCallback()`, or just happenstance during development?

Reply:
Ah - this was me messing around to see if I could fix the callback before we found that bug Friday afternoon. I'm happy to move it back to match the others, couldn't find any functional difference with this change.

Reply:
I don't think it matters much either way. I just wanted to be sure there wasn't something you found about the ordering of these mattering :sweat_smile: